config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
this file should probably not be in here, at least not with this content

Reply:
Aaaaand... it's gone